documentation/code-repos.md

```diff
@@ -10,70 +10,100 @@ Almost all HOCS code is open-source and publicly available on GitHub.
 
 ### Core components
 [hocs-audit](https://github.com/UKHomeOffice/hocs-audit)
+Receives audit event messages from an SQS queue for persistent storage.
 
 [hocs-case-creator](https://github.com/UKHomeOffice/hocs-case-creator)
+Reads a message containing a JSON payload from an AWS SQS Queue, and creates a cases with associateed data.
 
 [hocs-casework](https://github.com/UKHomeOffice/hocs-casework)
+Handles interactions with reference data specific to cases.
 
 [hocs-data 🔒️ *(to be deprecated)*](https://github.com/UKHomeOffice/hocs-data)
+The CS schema.
 
 [hocs-data-wcs 🔒️ *(to be deprecated)*](https://github.com/UKHomeOffice/hocs-data-wcs)
+The WCS schema.
 
 [hocs-docs](https://github.com/UKHomeOffice/hocs-docs)
+Handles interactions with documents stored in S3.
```

Comment:
Worth mentioning that this also handles the interactions with conversion and virus checking?